main.js

```diff
@@ -0,0 +1,112 @@
+
+function justDoIt (name){
+    return name + "justDoIt";
+}
+console.log(justDoIt+" "+("Rawan"));
+
+//part1 
+function bigOrSmallString (name){ 
+    if (name.length>10){ 
+        return name + "word is loooooong";
+        console.log("word is loooooong")
+
+    }else{
+        return name = "This word is short";
+        console.log("This word is loooooong!")
+    }
+}
+//part 2 
+function oddOrEvenString (string){
+    if (string % 2==o) {
+    return "This string length is TOTALLY even!";
+    console.log("This string length is TOTALLY even!")
+    }else{
+        "This string's length is odd"
+        console.log(This string's length is odd..")
```

Comment:
you forgot to add the `"` at the start of your string which created this crazy red highlight